Docker and Docker Compose update for Windows and new documents for Linux (#607)
OctopusDeploy/docs: docs/installation/octopus-in-container/octopus-server-container-windows.md

---
title: Octopus Deploy Server Container Windows
description: An Octopus Deploy Server instance can be run directly from within a container.
position: 1
---

> If you're looking to run an Octopus Deploy Linux container, please refer to the [Octopus Deploy Server Container Linux](/docs/installation/octopus-in-container/octopus-server-container-linux.md) documentation.

Reviewer: slight tweak:
```suggestion
If you want to run an Octopus Deploy Linux container, please refer to the [Octopus Deploy Server Container Linux](/docs/installation/octopus-in-container/octopus-server-container-linux.md) documentation.
```

|  Name       |    |
| ------------- | ------- |
|**DB_CONNECTION_STRING**|Connection string to the database to use|

> |**MASTER_KEY**|The master key to use to connect to an existing database. If not supplied, and the database does not exist, it will generate a new one. If the database does exist, this is mandatory|

Reviewer: ```suggestion
|**MASTER_KEY**|The master key to use to connect to an existing database. If not supplied, and the database does not exist, it will generate a new one. This is mandatory if the database exists|
```

> |**OCTOPUS_SERVER_BASE64_LICENSE**|Your License key for Octopus Deploy. If left empty, it will try and create a free license key for use

Reviewer: ```suggestion
|**OCTOPUS_SERVER_BASE64_LICENSE**|Your license key for Octopus Deploy. If left empty, it will try and create a free license key for use
```
